.github/dependabot.yml

```diff
@@ -0,0 +1,19 @@
+version: 2
+updates:
+  # Maintain dependencies for GitHub Actions
+  - package-ecosystem: "github-actions"
+    directory: "/"
+    schedule:
+      interval: "weekly"
+      time: '4:00'
+    open-pull-requests-limit: 5
+    rebase-strategy: "disabled"
+
+  # Maintain dependencies for Go
+  - package-ecosystem: "gomod"
+    directory: "/"
+    schedule:
+      interval: "daily"
+      time: '4:00'
+    open-pull-requests-limit: 5
+    rebase-strategy: "disabled"
```

Comment:
I know composer has the same config; but 5 PRs which all change `go.mod` sounds like a recipe for rebase issues. I wonder if it'll be more user-friendly if it has a limit of 1 open PR, but then checks every 4 hours.

Reply:
I don't think changing the schedule is needed. From [this](https://github.com/osbuild/osbuild-composer/pull/1724#issuecomment-913013581) comment from Ondřej, I guess dependabot stores the PR on a queue whenever there are more the the configured limit.

I'll change the open PR limit now.